### ISSUE TITLE
Why TF loss function gives different results than Pytorch?

### ISSUE THREAD

**Issue by chrismaliszewski** [tensorflow/tensorflow#43997]
I'm trying to figure out why TF binary_cross_entropy gives different results than Pytorch's binary_cross_entropy.

https://colab.research.google.com/drive/1hzFEI05_hKNRV4gE1FFDumd1ygN3LVsQ?usp=sharing

Anyone?

**Comment by ravikyram** [comment]
@chrismaliszewski 

Please, use` tf.keras.losses.BinaryCrossentropy`as loss function so you can see almost same results with TF and Pytorch.
Please, find the gist [here](https://colab.research.google.com/gist/ravikyram/90d4e8364e43be17ebf449a61826ebaf/untitled452.ipynb) for your reference.Thanks!

**Comment by chrismaliszewski** [comment]
Thank you, @ravikyram.
